pattern-1/dockerfiles/mysql/build.sh

```diff
@@ -18,19 +18,21 @@
 #
 # ------------------------------------------------------------------------
 
+set -e
+
 # Product Context
 product=wso2ei
 productVersion=6.1.1
 patternNumber=1
 # MySQL parameters
-rdbms=mysql
+database=mysql
 sqlVersion=5.5
 # Container Cluster Manager Info
 platform=kubernetes
 # Image Info
-repository=${product}-${platform}-pattern${patternNumber}-${rdbms}
+repository=${product}-${platform}-pattern${patternNumber}-${database}
 tag=${sqlVersion}
```

Comment:
Shouldn't this be termed as rdbms since what we are building here is a relational database management system container runtime ?

Reply:
Yes, I think it make sense! Let me revert it back. Thanks!